book/introduction/backend-template.rst

```diff
@@ -1,15 +1,29 @@
-Backend VS Templates
-====================
+Separation of Concerns
+======================
 
-As in any CMS Sulu completely separates content from design and allows
-template based content rendering. But due to the fact that usability, web
+As most of the modern Content Management Systems 
+Sulu completely separates content from design. It urges
+template based content rendering. Due to the fact that usability, web
```

Comment:
I would suggest we always use as much of the 80 characters per line as possible, and don't split lines if not necessary.